getting-started/exploring-forge.md

```diff
@@ -0,0 +1,109 @@
+# Exploring Timescale Forge
+
+Welcome to Timescale Forge! Timescale Forge combines the power and reliability 
+of TimescaleDB with a fully-managed, cloud-native experience that is easy to 
+start and less expensive to operate.
+
+This tutorial will walk you through setting up your Timescale Forge account and
+completing your first tutorial project.
+
+### Step 1: Create a Timescale Forge account
+
+Provide your full name, email address, and a strong password to start:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/forge_images/timescale-forge-signup-page.png" alt="Sign up for Timescale Forge"/>
+
+You will need to confirm your account by clicking the link you receive via 
+email. If you do not receive this link, please first check your spam folder 
+and, failing that, please [contact us][contact-timescale].
+
+### Step 2: Create your first service
+
+ After you complete account verification, you can visit the 
+ [Timescale Forge console][forge-console] and login with your credentials.
+
+ To begin, click 'Create service'.
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/forge_images/timescale-forge-service-creation.png" alt="Set up a Timescale Forge service"/>
```

Comment:
Please check if this location is consistent with how we have rest of content "images/docs/forge_images"?  But we can fix that in a subsequent PR.

cc: @tedsczelecki @solugebefola